README.md

```diff
@@ -48,24 +41,69 @@ docker-compose -f ./babel-local-dev/docker-compose.yml up
 
 In your browser:
 
-* http://localhost:8080/Search/Home
-* http://localhost:8080/cgi/pt?id=test.pd_open
+* catalog: `http://localhost:8080/Search/Home`
+* catalog solr: `http://localhost:9033`
+* full-text solr: `http://localhost:8983`
+
+imgsrv:
 
+* `http://localhost:8888/cgi/imgsrv/cover?id=test.pd_open`
+* `http://localhost:8888/cgi/imgsrv/image?id=test.pd_open&seq=1`
+* `http://localhost:8888/cgi/imgsrv/html?id=test.pd_open&seq=1`
+* `http://localhost:8888/cgi/imgsrv/download/pdf?id=test.pd_open&seq=1&attachment=0`
```

Comment:
My original vision was that you could proxy to the catalog fcgi app from so everything would run on the same port. If that's not a useful objective, we'll have to modify `common` to have known ports for catalog/babel.

Reply:
We can probably do that...

Reply:
This is working now.